File: app/src/main/java/com/starostinvlad/rxeducation/VideoScreen/Player.java
package com.starostinvlad.rxeducation.VideoScreen;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import okhttp3.Cookie;

public class Player {

    @SerializedName("name")
    @Expose
    private String name;
    @SerializedName("player")
    @Expose
    private String player;

    Cookie cookie;

    public Cookie getCookie() {
        return cookie;
    }

    public void setCookie(Cookie cookie) {
        this.cookie = cookie;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getPlayer() {
        return player;
    }

    public void setPlayer(String player) {
        this.player = player;
    }

}

File: app/src/main/java/com/starostinvlad/fan/VideoScreen/Player.kt
package com.starostinvlad.fan.VideoScreen

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName
import okhttp3.Cookie
import kotlin.Throws

class Player {
    @SerializedName("name")
    @Expose
    var name: String? = null

    @SerializedName("player")
    @Expose
    var player: String? = null
    var cookie: Cookie? = null
}